pkgs/tools/admin/ejson2env/default.nix

```
@@ -20,6 +20,25 @@ buildGoModule rec {
   ];
 
   passthru.updateScript = nix-update-script { };
+  passthru.tests = {
+    version = testers.testVersion { package = ejson2env; };
+    decryption = runCommand "check-ejson2env.sh" {} ''
```

This is better in a separate file,  say `./tests/decryption/default.nix`:

```suggestion
    decryption = callPackage ./tests/decryption { };
```